lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -242,26 +233,18 @@
             this.setSubmitButtonEnabled( false );
 
             // Load the template for the webcam into the DOM
-            renderedHtml = _.template( $( this.template ).html(), {} );
+            renderedHtml = _.template(
+                $( this.template ).html(),
+                { backendName: this.backend.name }
+            );
             $( this.el ).html( renderedHtml );
```

Comment:
If update template as per comment re. template should be able to update these 5 lines with:
this.$el.html( this.template({ backendName: this.backend.name }) );